conans/client/tools/files.py

```diff
@@ -224,6 +225,21 @@ def _manage_text_not_found(search, file_path, strict, function_name, output):
         return False
 
 
+@contextmanager
+def add_permissions(file_path, permissions):
```

Comment:
Make it private, I don't want to expose more tools to users (even if not documented) that we don't have a strong evidence that they are very necessary an demanded by users

Reply:
changed